namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-preprod/00-namespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sentence-planning-preprod
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "preprod"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/application: "sentence-planning"
+    cloud-platform.justice.gov.uk/owner: "Sentence Planning Team: michael.willis@digital.justice.gov.uk"
```

Comment:
This needs to be a team email address, not an individual.

Reply:
I fixed this, we have a team email here now